### PR TITLE
docs: add pg wire insert examples in Go, Java, Python and C

### PR DESCRIPTION
**Pull request by jaugsburger**

### CODE REVIEW

**Review by bluestreak01 — APPROVED** [review]

File: docs/develop/insert-data.md [hunk 1]
```diff
@@ -409,6 +413,136 @@ start()
...
+const (
+  host     = "localhost"
+  port     = 8812
+  user     = "admin"
+  password = "quest"
+  dbname   = "qdb"
+)
+
+func main() {
+    connStr := fmt.Sprintf("host=%s port=%d user=%s password=%s dbname=%s sslmode=disable", host, port, user, password, dbname)
```
Comment (mpsq) [inline docs/develop/insert-data.md:436]: Would it be possible to format the code with go-fmt?

File: docs/develop/insert-data.md [hunk 2]
```diff
@@ -409,6 +413,136 @@ start()
...
+
+    db, err := sql.Open("postgres", connStr)
+  if err != nil {
+    panic(err)
+  }
+  defer db.Close()
+
+  //currently we do not support queries with named variables in GO
+  rows, err := db.Query("insert into x values ('TRUCK')")
```
Comment (mpsq) [inline docs/develop/insert-data.md:445]: could you run the query:
`INSERT INTO test VALUES ('abc', 123`

For consistency with the other examples?

File: docs/develop/insert-data.md [hunk 3]
```diff
@@ -409,6 +413,136 @@ start()
...
+
+class App {
+  public static void main(String[] args) throws SQLException {
+    Properties properties = new Properties();
+    properties.setProperty("user", "admin");
+    properties.setProperty("password", "quest");
+    properties.setProperty("sslmode", "disable");
+
+    final Connection connection = DriverManager.getConnection("jdbc:postgresql://localhost:8812/qdb", properties);
+    try (PreparedStatement preparedStatement = connection.prepareStatement("insert into x (vehicle) values (?)")) {
```
Comment (mpsq) [inline docs/develop/insert-data.md:476]: Same as previous remark

File: docs/develop/insert-data.md [hunk 4]
```diff
@@ -409,6 +413,136 @@ start()
...
+}
+int main() {
+    PGconn *conn = PQconnectdb(
+            "host=localhost user=admin password=quest port=8812 dbname=testdb");
+    if (PQstatus(conn) == CONNECTION_BAD) {
+        fprintf(stderr, "Connection to database failed: %s\n",
+                PQerrorMessage(conn));
+        do_exit(conn);
+    }
+    PGresult *res = PQexec(conn, "INSERT INTO x VALUES ('BIKE');");
```
Comment (mpsq) [inline docs/develop/insert-data.md:511]: Again

File: docs/develop/insert-data.md [hunk 5]
```diff
@@ -409,6 +413,136 @@ start()
...
+```python
+import psycopg2
+try:
+    connection = psycopg2.connect(user="admin",
+                                  password="quest",
+                                  host="127.0.0.1",
+                                  port="8812",
+                                  database="qdb")
+    cursor = connection.cursor()
+    postgreSQL_select_Query = "INSERT INTO x VALUES ('LORRY')"
```
Comment (mpsq) [inline docs/develop/insert-data.md:533]: Same comment here, using the same query would be nice

File: docs/develop/query-data.md [hunk 6]
```diff
@@ -195,6 +195,8 @@ func main() {
 	checkErr(err)
 	defer db.Close()
 
+
+    //currently we do not support queries with named variables in GO
```
Comment (mpsq) [inline docs/develop/query-data.md:199]:
```suggestion
    // Currently, we do not support queries with bind parameters in Go
```

**Review by mpsq — CHANGES_REQUESTED** [review]

**Pull request merged**